readme.md

```diff
@@ -1,28 +1,484 @@
 #ES6
 
+## Learning Objectives
+
+- Compare/contrast features of ES5 and ES6
+- Explain when to use var vs let vs const
+- Use default parameters and arrow functions
+- Use template literals to interpolate variables and strings
+- Compare ES6 classes w/ ES5 prototypes
+- Use import/export syntax with modules
+
 ## A Brief History of JavaScript
 
+The JavaScript standard is referred to as EcmaScript.
+
+I like to think of EcmaScript as the language, and JavaScript an implementation of that language.
+
+ES3 was the first widespread use of the language. Unfortunately, ES4 never came out.
+
+In 2009, ES5 was finalized and it's what we've been writing in class.
+
+2015 brought ES6, published as a standard on June 17th, 2015.
+
+### Why now?
+
+Many plugins, frameworks and modules still use ES5, as browser support for
+the new version of the language is still limited [ahem... ie.](http://caniuse.com/#search=es6)
+
+Later today and tomorrow, we'll be working with a framework called React, and many
+React tutorials (and our lesson plans) have adopted the new syntax and features
+of ES6. Today is all about exploring those features and getting comfortable with
+the new syntax.
+
```

Comment:
Are you planning on just having them use ES6 in the browser? Might be appropriate to talk about some of the tooling out there to allow you to use ES6 now anywhere with tools like Babel.